TCC.Lib/CommandHelper.cs

```diff
@@ -96,7 +97,7 @@ public static bool IsIgnorable(string line)
             "-seed-ecb                  -seed-ofb                  "
         };
 
-        public static CommandResult Run(this string command, string workingDirectory, CancellationToken cancellationToken,
+        public static Task<CommandResult> Run(this string command, string workingDirectory, CancellationToken cancellationToken,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3776)